src/main/groovy/com/bmuschko/gradle/docker/tasks/image/Dockerfile.groovy

```diff
@@ -373,6 +373,30 @@ class Dockerfile extends DefaultTask {
         }
     }
 
+    static abstract class MapInstruction implements Instruction {
+        final Object command
+
+        MapInstruction(Map<String, String> command) {
+            this.command = command
+        }
+
+        MapInstruction(Closure command) {
```

Comment:
Again, I just went for consistency here.  I'm not using the closure-constructor form in my `LabelInstruction` subclass, but I see that the abstract classes are generally built to support this even if many of the implementation subclasses make no use of it.